### ISSUE TITLE
Real time isn't

### ISSUE THREAD

**Issue by jakebeal** [DanySK/alchemist#11]
When I tell the alchemist simulator to run in "real-time", it runs faster than real-time.
In addition, the "user controlled" throttle should relate to real-time, but doesn't currently appear to.

**Comment by DanySK** [comment]
As explained [here][UI], real time mode currently forces a minimum of 25 events per second. Is this the limit you are hitting?
Also, Protelis events by default are scheduled to run in the first 10% of the chosen average time, which means that if there are no other events, such as movements, the first 10% of a second will be run in real time, and then the remaining 90% will be run in a single shot. Do you run at approximately 10x real time?
This was a setting we believed to be reasonable. We may want to open up this window to 100%.
We may also want to change the behaviour of the real time mode.

[UI]: https://danysk.github.io/alchemist/pages/tutorial/swingui/

**Comment by jakebeal** [comment]
Yes, I'm seeing about 10x speedup, and might or might not be hitting the minimum too.
I think that real-time should mean real-time.  If it's not going to be real-time, it should be called something else.
